_sips/process-specification.md

```diff
@@ -0,0 +1,315 @@
+  rejection.
+- Keep: the proposal remains in the Design Stage for further iterations. The 
+  reviewers will communicate on the PR the current concerns raised by the 
+  Committee.
+
+In order to be accepted for implementation and advance to the next stage, a SIP
+must gather strictly more than 50% of "Advance" votes among the members present
+(this means that an abstention is equivalent to "Do not advance" for this 
+purpose, biasing the process in favor of the status quo). The PR for the SIP is
+then also Merged at this point.
+
+If there was no strict majority in favor of "Advance", a second vote between 
+Reject and Keep will be used. A proposal needs more than 50% "Reject" votes to
+be rejected in that case. Otherwise, it is kept.
+
+### Implementation Stage
+
+Once in the implementation stage, the Committee is not concerned with the SIP
+anymore, until new concerns are discovered or until the implementation is ready.
+The SIP is now a recommendation for the compiler team or any other individual or
+group of people to provide an implementation of the proposal, as a pull request
+to the Scala 3 compiler repository. There is no set timeline for this phase.
+
+An implementation will be reviewed by the compiler team, and once the
+implementation is deemed good enough, it can ship as an Experimental feature in
+the following practical release of the compiler.
+
+The implementers may hit challenges that were not foreseen by the Committee. 
+Early users may also provide feedback based on practical experience with the 
+Experimental feature. This feedback can be sent back to the Committee by 
+implementers. This is done with a PR to the SIP repository, amending the 
+previously merged SIP document or raising questions for challenges. In that
+case, the SIP Author and Reviewers will work together with the implementers to
+address the feedback. This is again an iterative process. Reviewers may merge
+changes to the proposal at their discretion during this phase.
+
+Once the implementation is deemed stable, including appropriate tests, the
+implementers and reviewers can schedule the SIP to the next Committee meeting
+for final approval. Once again, a SIP needs to gather strictly more than 50% 
+"Accept" votes to be Completed. If that is not achieved, it may likewise be sent 
+back for refinements, or be rejected, with the same rules as in the "Design 
+Stage -- Vote" section.
+
+### Completed Stage
+
+Once a SIP is accepted for shipping, it will be enabled by default (as 
+non-Experimental) in the next practical Minor release of the language.
+
+From this point onwards, the feature is stable and cannot be changed anymore. 
+Any further changes would have to come as an entirely new SIP.
+
+## The SIP Committee
+
+The current committee members are:
+
+- Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+- Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), Scala Center
```

Comment:
Ah ah!

For anyone reading this at this moment, this is of course a placeholder. That list must be expanded. ;)

Reply:
Or just edit it to read:
```
The remaining committee members are:
```